docs/reference/client-config.md

```diff
@@ -113,6 +118,48 @@ Remote server example:
 	</tr>
 </table>
 
+## trust_pinning section (optional)
+
+The `trust_pinning` specifies how to bootstrap trust for the root of a
+Notary client's trusted collection.
+
+This section is optional, Notary will use TOFU over HTTPS by default and
+trust certificates in the downloaded root file.
+
+In this section, one can provide specific certificates to pin to, or a CA
+to pin to as a root of trust for a GUN.
+
+Only one trust pinning option can be used per GUN, and any validation failure
```

Comment:
This seems to imply that if more than one option is given, the parsing the options will fail, but the code seems to prioritize one over the other.  If so, would it make sense to specify the order of preference of options here?  The design doc is actually pretty clear and that state diagram seems pretty useful - would it make sense to include it in the docs somewhere?